aiohttp_session/__init__.py

```diff
@@ -143,6 +159,16 @@ def max_age(self):
     def cookie_params(self):
         return self._cookie_params
 
+    def get_session_data(self, session):
```

Comment:
I believe it's private method, it should be named `_get_session_data` (starting with underscore).